docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
+that can be packaged in a container image layer , a `git` repository or any other content source that the
+[plain bundle provisioner](../provisioners/plain.md) supports.
+
+Supported source types for a plain bundle currently include the following:
+
+* A directory in a [container image](../sources/image.md)
```

Comment:
Do we want to call the directory a bundle instead of the complete image, git repository or configMap that happen per the specifications to have a directory containing the manifests?
I find the match bundle-directory a bit confusing, especially as it is not a directory location which is passed to the provisioner through Bundles or BundleDeployments but an image, git repo location

Reply:
I would *personally* prefer we keep the collection of manifests what we call a "bundle". I think it is totally reasonable that we provide a way for a user to specify where their bundle exists in the source but keep a sane default if one is unprovided. 

The bundle-directory is just an opinionated concept embedded within RukPak's bundle sourcing logic for specific source types and is something that we can progressively make less opinionated in follow ups (if we decided that is what we want to do)